style.css

```
@@ -0,0 +1,643 @@
+body {
+    position: relative;
+    width: 1366px;
+    margin: 0 auto;
+    padding: 0;
+    font-family: Golos Text;
+    color: rgba(49, 49, 49, 1);
+    background-color: rgba(255, 255, 255, 1);
+
+}
+
+.background {
+    background-color: rgba(233, 222, 212, 1);
+    width: 1366px;
```

```suggestion
    width: 100%;
```

опять же, это про резиновую вёрстку